internal/db/schema/migrations/oss/postgres/76/01_deleted_tables_and_triggers.up.sql

```diff
@@ -0,0 +1,247 @@
+-- Copyright (c) HashiCorp, Inc.
+-- SPDX-License-Identifier: MPL-2.0
+
+begin;
+  create table target_deleted (
+    public_id wt_public_id primary key,
+    delete_time wt_timestamp
```

Comment:
I think this should be `not null`?
```suggestion
    delete_time wt_timestamp not null
```